test-foundry/TestInvariantHeapOrdering.t.sol

```diff
@@ -0,0 +1,72 @@
+// SPDX-License-Identifier: GNU AGPLv3
+pragma solidity ^0.8.0;
+
+import "ds-test/test.sol";
```

Comment:
Why this import? I think foundry allows you to directly import it now

Reply:
Wdym I don't understand. I need it for `DSTest`.

Reply:
Yes but just import import `"@forge-std/Test.sol";` then because `DSTest` is imported by it

Reply:
done